### ISSUE TITLE
DOCX File Not Opening in Microsoft Word When Using Margins Configuration

### ISSUE THREAD

**Issue by maxdinech** [TurboDocx/html-to-docx#35]
Hello,

I encountered an issue where DOCX files generated using the `html-to-docx` library fail to open in Microsoft Word. The error message displayed by Word is:

```
Word encountered an error trying to open the file.
```

After investigating the problem, I discovered that the issue is related to the `margins` object in the `documentOptions` configuration. When I remove the `margins` object, the DOCX file is generated correctly and opens without any issues in Word.

**Steps to Reproduce:**
1. Use the following code to generate a DOCX file:
    ```javascript
    import HTMLtoDOCX from '@turbodocx/html-to-docx';

    const htmlContent = `
      <html>
        <body>
          <h1>Hello, World!</h1>
          <p>This is a test document.</p>
        </body>
      </html>
    `;

    const documentOptions = {
      margins: {
        top: 1000,
        right: 1000,
        bottom: 1000,
        left: 1000,
      },
      font: 'Calibri',
    };

    const docxBuffer = await HTMLtoDOCX(htmlContent, '', documentOptions, '');
    ```
2. Attempt to open the generated DOCX file in Microsoft Word.

**Observed Behavior:**
- Word displays an error message and fails to open the file.

**Expected Behavior:**
- Word should open the DOCX file without any errors.

**Workaround:**
- Removing the `margins` object from the `documentOptions` allows the DOCX file to open correctly in Word.

**Example of Working Code (without margins):**
```javascript
import HTMLtoDOCX from '@turbodocx/html-to-docx';

const htmlContent = `
  <html>
    <body>
      <h1>Hello, World!</h1>
      <p>This is a test document.</p>
    </body>
  </html>
`;

const documentOptions = {
  font: 'Calibri',
};

const docxBuffer = await HTMLtoDOCX(htmlContent, '', documentOptions, '');
```

**Environment:**
- Library Version: `@turbodocx/html-to-docx` 1.9.3
- Node.js Version: 22.2.0
- Operating System: macOS

**Comment by nicolasiscoding** [comment]
Thank you for submitting, we will take a look

**Comment by nicolasiscoding** [comment]
@maxdinech fixed in v1.9.4
